docker/python/manylinux2010/Dockerfile

```diff
@@ -6,8 +6,7 @@
 # the Apache License 2.0.  The full license can be found in the LICENSE file.
 #
 
-# https://github.com/pypa/manylinux/issues/428#issuecomment-774729597
-FROM quay.io/pypa/manylinux2010_x86_64:2021-02-06-3d322a5
+FROM quay.io/pypa/manylinux2010_x86_64
```

Comment:
As per #1820 and others, I want to avoid these changes until they're reproducible or removable.